iroh-bitswap/src/lib.rs

```diff
@@ -302,7 +302,7 @@ impl<S: Store> Bitswap<S> {
         record!(BitswapMetrics::MessageBytesIn, message.encoded_len() as u64);
         // TODO: Handle backpressure properly
         if let Err(err) = self.incoming_messages.try_send((peer, message)) {
-            warn!(
+            debug!(
```

Comment:
this should stay a `warn` if this happens it means we are under too much load.

Reply:
Done

Reply:
> I see this message every time without doing anything special

yes because as soon as you announce you are using bitswap, the network hammers you :'(